riscv/csrs.cc

```diff
@@ -862,7 +862,7 @@ satp_csr_t::satp_csr_t(processor_t* const proc, const reg_t addr):
 void satp_csr_t::verify_permissions(insn_t insn, bool write) const {
   base_atp_csr_t::verify_permissions(insn, write);
   if (get_field(state->mstatus->read(), MSTATUS_TVM))
-    require(state->prv >= PRV_M);
+    require(state->prv == PRV_M);
```

Comment:
The commit message of "fix permission check" is misleading, even alarming, because it makes it sound like there was a bug with this CSR's permission checks. But this commit doesn't actually change any behavior. It clarifies a confusing line of code, that's all.